extraConfigDpu.py

```diff
@@ -218,9 +215,6 @@ def ExtraConfigDpuHost(cc: ClustersConfig, cfg: ExtraConfigArgs, futures: dict[s
     vendor_plugin = init_vendor_plugin(h, node.kind or "")
     if isinstance(vendor_plugin, IpuPlugin):
         vendor_plugin.build_push(lh, imgReg)
-    else:
-        vendor_plugin.build_push_start(lh, client, imgReg)
-        wait_vsp_ds_running(client)
 
```

Comment:
@thom311 FYI removing this step now, since Marvell VSP deployment is no longer a "manual" step

Reply:
ok. That's fine. I have a patch that does basically the same.